<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>微信爬虫组件</string>
  </property>
  <widget class="QPlainTextEdit" name="cookieEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>691</width>
     <height>231</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>添加cookie...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>510</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="bizEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>691</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>添加biz...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="testButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>510</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="logEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>691</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
